### DISCUSSION TITLE
Distribution of relevant and irrelevant papers

### DISCUSSION THREAD

**Discussion by willekekit** [asreview/asreview#934] [Q&A]
On the ASReview website there is an example of how screening graphs may look like (https://asreview.nl/blog/asreview-class-101/#stop-screening). During our screening, we noticed straight away that there were a lot of relevant papers, which looks different from the example on the website (where there is a bigger learning curve; more irrelevant papers at the start).

After screening 280 papers (=5% of dataset), we have included 181 papers and still we have a lot of relevant paper coming through. From the abstracts  we've screened so far, it seems unlikely that many papers will end up irrelevant after fulltext screening. 

Does this mean our research question is too broad? Should we expect a graph to always look somewhat along the lines of what it looks like in the example on the website?

**Comment by Rensvandeschoot** [comment]
Every project is different :-) 
If you find higher rates of relevant papers compared to the example on the website, this is only good news! It probably means the papers used to train the first iteration of the model (prior knowledge) are very useful for the machine to 'understand' what kind of papers you are looking for. Also, probably you have very clear inclusion/exclusion criteria so that every iteration you provide the machine with high-quality input.

At some point you want the % of relevant papers to decrease and become 0% for many iterations. See for example the curve for this project on a systematic review on [AI-aided systematic review tools](https://asreview.nl/blog/project/evaluating-the-performance-of-active-learning-within-systematic-reviews/).

**Reply by willekekit** [reply]
Thank you Rens. 

It seems we are going to end up with over 200 papers to review (which is, in our eyes, not feasible), because the curve repeats the 0-91 section till 280 several times. For each of our 3 outcome variables, there is a new 'block'. We are considering dropping one outcome variable now and specifying the study types to include.

<img width="233" alt="ASReview_progress_5%" src="https://user-images.githubusercontent.com/97950297/149963517-12bae8c2-8cf8-40a9-b6bc-93226eb2c250.png">

**Reply by Rensvandeschoot** [reply]
Including so many papers has nothing to do with Active Learning :-) 
It are your inclusion/exclusion criteria :-(
Also when you would have the old-fashioned way, you would have faced the issue of too many papers to read. A solution might be to reconsider your inclusion and exclusion criteria (be more strict), and/or directly make the full-text decision. If the DOI is part of the dataset, you can simply click on the hyperlink [provided by ASReview LAB](https://asreview.readthedocs.io/en/latest/features/screening.html#doi), or try the [Scholar Google button](https://asreview.nl/blog/tools-that-work-well-with-asreview-google-scholar-button/).  

Alternatively, it might be there are indeed so many relevant papers to include, which is a result of your study and not an issue at all, but a reason to celebrate: your topic is of great importance!! (and you still need to find a way to summarize all studies)
